frontend/src/utils.js

```diff
@@ -0,0 +1,5 @@
+function setIntersection(setA, setB) {
+  return new Set([...setA].filter(x => setB.has(x)))
+}
+
+export { setIntersection }
```

Comment:
trailing newline needed.. we'll set up a linter :p

Reply:
Hehe okay we can do that. My editor is not setup on this machine srry